lib/improver/tests/nbhood/use_nbhood/test_CollapseMaskedNeighbourhoodCoordinate.py

```diff
@@ -102,35 +117,36 @@ def setUp(self):
         self.plugin = CollapseMaskedNeighbourhoodCoordinate("topographic_zone",
                                                             self.weights_cube)
 
+    def test_basic(self):
+        """Test that a cube is modified by this function"""
```

Comment:
Recommended: "Test weights_cube is still a cube after the function call"

Reply:
Changed